src/run/TroubleshootAnalyses.ts

```diff
@@ -161,4 +165,19 @@ export default class TroubleshootAnalyses {
     private documentationLink (documentationFileName: string) {
         return `https://www.github.com/companieshouse/chs-dev/blob/main/docs/${documentationFileName}`;
     }
+
+    /**
+ * Filters out `ProxiesConfiguredCorrectlyAnalysis` analysis tasks if iBoss (a network security service) is enabled.
+ * @param {AnalysisTask[]} analysisTasks - The list of analysis tasks to be processed.
+ * @returns {AnalysisTask[]} - The filtered list of analysis tasks with proxy checks removed if iBoss is enabled.
+ */
+    private handleIbossCheck (analysisTasks: AnalysisTask[]): AnalysisTask[] {
+
+        if (isIbossEnabled()) {
+            analysisTasks = analysisTasks.filter(
+                task => !(task instanceof ProxiesConfiguredCorrectlyAnalysis)
+            );
+        }
+        return analysisTasks;
```

Comment:
Rather than modify the overall TroubleshootAnalyses class I'd probably amend the AnalysisTask responsible for raising them - this is not likely to be a permanent fixture so likely to be irrelevant in time